dask_sphinx_theme/layout.html

```diff
@@ -1,5 +1,33 @@
 {%- set favicon_url = favicon_url or ('_static/' + ('images/favicon.svg')) %} {%
-extends "sphinx_book_theme/layout.html" %} {%- block content %}
+extends "sphinx_book_theme/layout.html" %} {% block extrahead %}
```

Comment:
Would it be possible to add this to the footer instead of the head? Putting it in the head will block the page from loading until the script is done.

Reply:
Yup, I can move it down! I was following the [GTM instructions](https://developers.google.com/tag-platform/tag-manager/web), but at least [one person on the internet](https://www.analyticsmania.com/post/google-tag-manager-head-or-body/) claims moving it down won't affect analytics